charts/aws-ebs-csi-driver/templates/node/daemonset-windows.yaml

```diff
@@ -37,53 +38,63 @@ spec:
       nodeSelector:
         kubernetes.io/os: windows
         {{- with .Values.node.nodeSelector }}
-        {{- toYaml . | nindent 8 }}
+          {{- toYaml . | nindent 8 }}
         {{- end }}
-      serviceAccountName: {{ .Values.node.serviceAccount.name }}
-      priorityClassName: {{ .Values.node.priorityClassName | default "system-node-critical" }}
+      serviceAccountName: {{ include "aws-ebs-csi-driver.node.serviceAccountName" . }}
+      priorityClassName: {{ default "system-node-critical" .Values.node.priorityClassName }}
       tolerations:
-        {{- if .Values.node.tolerateAllTaints }}
+      {{- if .Values.node.tolerateAllTaints }}
         - operator: Exists
-        {{- else }}
+      {{- else }}
         - key: CriticalAddonsOnly
           operator: Exists
         - operator: Exists
           effect: NoExecute
           tolerationSeconds: 300
-        {{- end }}
-        {{- with .Values.node.tolerations }}
+      {{- end }}
+      {{- with .Values.node.tolerations }}
+        {{- toYaml . | indent 8 }}
+      {{- end }}
+      {{- with .Values.node.podSecurityContext }}
+      securityContext:
         {{- toYaml . | nindent 8 }}
-        {{- end }}
+      {{- end }}
       containers:
         - name: ebs-plugin
           image: {{ printf "%s:%s" .Values.image.repository (default (printf "v%s" .Chart.AppVersion) (toString .Values.image.tag)) }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
             - node
             - --endpoint=$(CSI_ENDPOINT)
             {{- with .Values.node.volumeAttachLimit }}
             - --volume-attach-limit={{ . }}
             {{- end }}
             - --logtostderr
-            - --v={{ .Values.node.logLevel }}
+          {{- with .Values.node.logLevel }}
+            - --v={{ . }}
+          {{- end }}
+          {{- with .Values.node.additionalArgs }}
+            - {{ toYaml . | nindent 12 }}
+          {{- end }}
           env:
             - name: CSI_ENDPOINT
-              value: unix:/csi/csi.sock
+              value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
```

Comment:
Are these paths different on different nodes?  Should this be configurable?

Reply:
This can be set to any path, but this and the volume mount path should be same.

Reply:
What I mean is that on one windows system could this be a different path than on another?  Should the user be able to specify/override this value?